<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicFlow — Плеер</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
    background-color: #121212;
    color: #FFFFFF;
    font-family: &quot;Segoe UI&quot;, &quot;Roboto&quot;, sans-serif;
}

QTabWidget::pane {
    border: 1px solid #333;
    border-radius: 8px;
}

QTabBar::tab {
    background: #1E1E1E;
    border: 1px solid #333;
    padding: 8px 16px;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
    color: #FFFFFF;
}

QTabBar::tab:selected {
    background: #3B82F6;
    color: #FFFFFF;
}

QTabBar::tab:hover {
    background: #2563EB;
}

QPushButton {
    background-color: #3B82F6;
    border-radius: 6px;
    padding: 6px;
    color: white;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #2563EB;
}

QPushButton:pressed {
    background-color: #1D4ED8;
}

QLineEdit {
    background-color: #1E1E1E;
    border: 2px solid #333;
    border-radius: 6px;
    padding: 6px;
    color: #FFFFFF;
}

QLabel {
    color: #FFFFFF;
}
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="nowPlayingLayout">
      <item>
       <widget class="QLabel" name="trackCoverLabel">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>320</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { background-color: #1e1e1e; border: 1px solid #333; border-radius: 12px; }</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="trackTitleLabel">
        <property name="styleSheet">
         <string notr="true">QLabel { font-size: 20px; font-weight: 600; }</string>
        </property>
        <property name="text">
         <string>—</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="controlsLayout">
        <item>
         <spacer name="controlsLeftSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="prevTrackButton">
          <property name="text">
           <string>⏮</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playPauseButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>▶️</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextTrackButton">
          <property name="text">
           <string>⏭</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="favoriteButton">
          <property name="text">
           <string>♡</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="controlsRightSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="volumeSlider">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>70</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <widget class="QWidget" name="recommendTab">
       <attribute name="title">
        <string>Рекомендации</string>
       </attribute>
       <layout class="QVBoxLayout" name="recommendLayout">
        <item>
         <widget class="QListWidget" name="recommendationList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="favoritesTab">
       <attribute name="title">
        <string>Избранное</string>
       </attribute>
       <layout class="QVBoxLayout" name="favoritesLayout">
        <item>
         <widget class="QListWidget" name="favoritesList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="uploadTab">
       <attribute name="title">
        <string>Загрузка</string>
       </attribute>
       <layout class="QHBoxLayout" name="uploadLayout">
        <item>
         <layout class="QVBoxLayout" name="uploadLeft">
          <item>
           <widget class="QLineEdit" name="songNameInput">
            <property name="placeholderText">
             <string>Название песни</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="chooseFileButton">
            <property name="text">
             <string>Выбрать песню</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="filePathLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="chooseAvatarButton">
            <property name="text">
             <string>Выбрать обложку</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="avatarPathLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uploadButton">
            <property name="text">
             <string>Загрузить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="uploadStatus">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="uploadCoverPreview">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>240</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel { background: qlineargradient(x1:0,y1:0,x2:1,y2:1, stop:0 #111, stop:1 #1e1e1e); border: 1px solid #333; border-radius: 12px; }</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
